server/sonar-web/src/main/js/apps/component-measures/home/reducer.js

```diff
@@ -31,7 +31,7 @@ const initialState = {
 };
 
 function groupByDomains (measures) {
-  const KNOWN_DOMAINS = ['Reliability', 'Security', 'Maintainability', 'Tests', 'Duplication', 'Size', 'Complexity'];
+  const KNOWN_DOMAINS = ['Reliability', 'Security', 'Maintainability', 'Coverage', 'Duplications', 'Size', 'Complexity'];
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Split this 121 characters long line (which is greater than 120 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=javascript%3ALineLength)